Source/Transforms/LMCHilbertTransform.cc

```diff
@@ -15,13 +15,40 @@ namespace locust
 
 
     HilbertTransform::HilbertTransform():
-        fbufferMargin( 25 ),
-		fbufferSize( 50 )
+        fbufferMargin( 50 ),
+		fbufferSize( 100 ),
+        originaldata(NULL),
+        SignalComplex(NULL),
+        FFTComplex(NULL),
+        hilbert(NULL),
+        fWindowName( "hanning" ),
```

Comment:
Can this be changed back to its default value of "rectangular", and instead move this new setting to the default config files in the hexbug repo?

Reply:
same for the buffer lengths? 50 is just too short

Reply:
yes, I would say that the modified user-specific values for buffer length parameters can best be specified in the hexbug config files.